code/scripting/api/objs/modelinstance.cpp

```diff
@@ -148,6 +148,29 @@ ADE_VIRTVAR(Orientation, l_SubmodelInstance, "orientation", "Gets or sets the su
 	return ade_set_args(L, "o", l_Matrix.Set(matrix_h(&smi->canonical_orient)));
 }
 
+ADE_VIRTVAR(Offset, l_SubmodelInstance, "vector", "Gets or sets the translated submodel instance offset", "vector", "Offset, or zero vector if handle is not valid")
```

Comment:
Since there is a difference between orientation and offset in there being a non-instance offset, I'd like a clarification here that this is _only_ the offset relative to the already existing submodel offset-to-model-center

Reply:
Ok.